UPGRADE-1.8.md

```diff
@@ -74,15 +84,53 @@
     +               guard:
     +                   authenticators:
     +                       - lexik_jwt_authentication.jwt_token_authenticator
+    + 
+            access_control:
+    +            - { path: "%sylius.security.new_api_admin_regex%/.*", role: ROLE_API_ACCESS }
+    +            - { path: "%sylius.security.new_api_shop_regex%/.*", role: IS_AUTHENTICATED_ANONYMOUSLY }
     ```
-
 1. Add `sylius_api.yaml` file to `config/routes/` directory:
 
     ```yaml
        sylius_api:
            resource: "@SyliusApiBundle/Resources/config/routing.yml"
            prefix: "%sylius.security.new_api_route%"
     ```
+1. Add `lexik_jwt_authentication.yaml` file to `config/packages/` directory:
+
+    ```yaml
+       lexik_jwt_authentication:
+         secret_key: '%env(resolve:JWT_SECRET_KEY)%'
+         public_key: '%env(resolve:JWT_PUBLIC_KEY)%'
+         pass_phrase: '%env(JWT_PASSPHRASE)%'
+    ```
+
+1. Add configuration in your `.env` file:
+
+    ```diff
+    +       ###> lexik/jwt-authentication-bundle ###
+    +       JWT_SECRET_KEY=%kernel.project_dir%/config/jwt/private.pem
+    +       JWT_PUBLIC_KEY=%kernel.project_dir%/config/jwt/public.pem
+    +       JWT_PASSPHRASE=cd093de0e84512a4087839ef05088ffe
```

Comment:
There should be probably information about generating these files and that `JWT_PASSPHRASE` value is an example